pie/scripts/app.py

```diff
@@ -41,6 +41,25 @@
 
 """
 
+# If we are running python pie/scripts/app.py
+#  We need to add pie to the python path
+try:
+    from pie.webapp import bind
+except ModuleNotFoundError as E:
+    if str(E) == "No module named 'pie'":
```

Comment:
Is this necessary for the pip packaging? I'd rather prefer not having to add stuff to their path. Basically, just letting it crash and tell them to run app.py from root directory.

Reply:
This is not necessary for pip, but when pip is not used.
If I remove that, instead of running `python pie/scripts/tag.py ____` (which mirrors the actual `python tag.py ___`), we'll have to type `python -m pie.scripts.tag` which I am fine with.

Reply:
yes I'd prefer that solution, then all this catch blocks can go.